src/components/tables/Captures/StatsHydrator.tsx

```diff
@@ -0,0 +1,44 @@
+import {
+    SelectableTableStore,
+    selectableTableStoreSelectors,
+} from 'components/tables/Store';
+import { useZustandStore } from 'context/Zustand/provider';
+import { useEffect } from 'react';
+import { SelectTableStoreNames } from 'stores/names';
+import { BaseComponentProps } from 'types';
+
+// Hydrator
+interface StatsHydratorProps extends BaseComponentProps {
+    selectableTableStoreName: SelectTableStoreNames;
+}
+
+const StatsHydrator = ({
```

Comment:
This is all still WIP as I am still sketching out code to decide how we want to do this.